lib/Index/Index.cpp

```diff
@@ -2155,6 +2155,12 @@ void index::indexSourceFile(SourceFile *SF, IndexDataConsumer &consumer) {
 
 void index::indexModule(ModuleDecl *module, IndexDataConsumer &consumer) {
   assert(module);
+  auto mName = module->getRealName().str().str();
```

Comment:
If I followed these through correctly, this ends up as a `std::string`. Any reason not to just pass the `StringRef` through?

Reply:
Fixed!